### ISSUE TITLE
Zfs hangs

### ISSUE THREAD

**Issue by F0x06** [zfsonlinux/zfs#3487]
Dear all, 

My storage server got freezed after few hours of intense I/O (through NFS) the only method to unblock it is to reboot the machine, anyone can help me ?
thanks in advance.

OS: ```Ubuntu 14.04.2 LTS ( 3.13.0-53-generic #89-Ubuntu SMP Wed May 20 10:34:39 UTC 2015 x86_64 x86_64 x86_64 GNU/Linux )```
ZFS version: ```v0.6.4-85_gc3520e7, ZFS pool version 5000, ZFS filesystem version 5 (git master)```
SPL version: ```v0.6.4-6_ga876b03 (git master)```
Memory:  ```16 GB``` 

This is my current pool setup:

```
cluster_1
    raidz2-0
        sda3 (6 TB)
        sdb  (6 TB)
        sdc  (6 TB)
        sdd  (6 TB)
        sde  (6 TB)
        sdf  (6 TB)
```

And the syslog stack trace:

```
Jun 12 02:32:12 storage-server kernel: [31051.688515] general protection fault: 0000 [#1] SMP
Jun 12 02:32:12 storage-server kernel: [31051.688628] Modules linked in: nfsv3 nfsv4 ib_iser rdma_cm iw_cm ib_cm ib_sa ib_mad ib_core ib_addr iscsi_tcp libiscsi_tcp libiscsi scsi_transport_iscsi snd_hda_codec_hdmi eeepc_wmi asus_wmi sparse_keymap mxm_wmi crc32_pclmul aesni_intel aes_x86_64 lrw dm_multipath gf128mul glue_helper scsi_dh ablk_helper ghash_clmulni_intel cryptd kvm coretemp crct10dif_pclmul intel_powerclamp x86_pkg_temp_thermal serio_raw snd_hda_codec_realtek i915 snd_hda_intel snd_hda_codec snd_hwdep snd_pcm drm_kms_helper mei_me snd_page_alloc drm snd_timer mei snd soundcore shpchp i2c_algo_bit video mac_hid acpi_pad wmi lp parport nfsd auth_rpcgss nfs_acl nfs lockd sunrpc fscache zfs(POX) zunicode(POX) zcommon(POX) znvpair(POX) spl(OX) zavl(POX) psmouse ahci r8169 libahci mii dm_mirror dm_region_hash dm_log
Jun 12 02:32:12 storage-server kernel: [31051.689967] CPU: 0 PID: 3369 Comm: z_rd_int/2 Tainted: P           OX 3.13.0-53-generic #89-Ubuntu
Jun 12 02:32:12 storage-server kernel: [31051.690056] Hardware name: ASUS All Series/Z97-K, BIOS 2604 05/20/2015
Jun 12 02:32:12 storage-server kernel: [31051.690122] task: ffff8800d278e000 ti: ffff8803fb154000 task.ti: ffff8803fb154000
Jun 12 02:32:12 storage-server kernel: [31051.690194] RIP: 0010:[<ffffffffa01d5d4b>]  [<ffffffffa01d5d4b>] zio_walk_parents+0x3b/0x70 [zfs]
Jun 12 02:32:12 storage-server kernel: [31051.690380] RSP: 0018:ffff8803fb155cf8  EFLAGS: 00010286
Jun 12 02:32:12 storage-server kernel: [31051.690434] RAX: fffd88023beb4ea0 RBX: ffff88029ad02760 RCX: 00000000001c0980
Jun 12 02:32:12 storage-server kernel: [31051.690503] RDX: ffff88029ad02860 RSI: 0000000000008000 RDI: ffff88029ad02760
Jun 12 02:32:12 storage-server kernel: [31051.690572] RBP: ffff8803fb155cf8 R08: ffff8803fd8cc000 R09: 0000000000000001
Jun 12 02:32:12 storage-server kernel: [31051.690641] R10: ffffffffa007a027 R11: ffffea000ef1b180 R12: ffff8800d278e000
Jun 12 02:32:12 storage-server kernel: [31051.690710] R13: 0000000000000000 R14: ffff88029ad02ab0 R15: ffff88029ad02a88
Jun 12 02:32:12 storage-server kernel: [31051.690780] FS:  0000000000000000(0000) GS:ffff88041fa00000(0000) knlGS:0000000000000000
Jun 12 02:32:12 storage-server kernel: [31051.690858] CS:  0010 DS: 0000 ES: 0000 CR0: 0000000080050033
Jun 12 02:32:12 storage-server kernel: [31051.690915] CR2: 00007f86ec5cd000 CR3: 0000000001c0e000 CR4: 00000000001407f0
Jun 12 02:32:12 storage-server kernel: [31051.690984] Stack:
Jun 12 02:32:12 storage-server kernel: [31051.691006]  ffff8803fb155d70 ffffffffa01d9a6c ffffffff81729642 ffff88029ad02760
Jun 12 02:32:12 storage-server kernel: [31051.691092]  ffff8803fb155d48 ffffffffa01d3db7 ffff88029ad02ab0 ffff88029ad02a88
Jun 12 02:32:12 storage-server kernel: [31051.691177]  0000000000000101 0000000000080000 ffff88029ad02760 ffff8800d278e000
Jun 12 02:32:12 storage-server kernel: [31051.691261] Call Trace:
Jun 12 02:32:12 storage-server kernel: [31051.691373]  [<ffffffffa01d9a6c>] zio_done+0x32c/0xe60 [zfs]
Jun 12 02:32:12 storage-server kernel: [31051.691435]  [<ffffffff81729642>] ? mutex_lock+0x12/0x2f
Jun 12 02:32:12 storage-server kernel: [31051.691566]  [<ffffffffa01d3db7>] ? zio_wait_for_children+0x87/0xa0 [zfs]
Jun 12 02:32:12 storage-server kernel: [31051.691708]  [<ffffffffa01d9fca>] zio_done+0x88a/0xe60 [zfs]
Jun 12 02:32:12 storage-server kernel: [31051.691767]  [<ffffffff81729642>] ? mutex_lock+0x12/0x2f
Jun 12 02:32:12 storage-server kernel: [31051.691898]  [<ffffffffa01d5948>] zio_execute+0xc8/0x180 [zfs]
Jun 12 02:32:12 storage-server kernel: [31051.691972]  [<ffffffffa007b200>] taskq_thread+0x1b0/0x360 [spl]
Jun 12 02:32:12 storage-server kernel: [31051.692040]  [<ffffffff8109ab20>] ? wake_up_state+0x20/0x20
Jun 12 02:32:12 storage-server kernel: [31051.692108]  [<ffffffffa007b050>] ? taskq_cancel_id+0x120/0x120 [spl]
Jun 12 02:32:12 storage-server kernel: [31051.692176]  [<ffffffff8108b6b2>] kthread+0xd2/0xf0
Jun 12 02:32:12 storage-server kernel: [31051.692228]  [<ffffffff8108b5e0>] ? kthread_create_on_node+0x1c0/0x1c0
Jun 12 02:32:12 storage-server kernel: [31051.692296]  [<ffffffff81733868>] ret_from_fork+0x58/0x90
Jun 12 02:32:12 storage-server kernel: [31051.692352]  [<ffffffff8108b5e0>] ? kthread_create_on_node+0x1c0/0x1c0
Jun 12 02:32:12 storage-server kernel: [31051.692416] Code: 48 85 c0 74 2b 48 8b 97 f8 00 00 00 48 8d 8f 00 01 00 00 48 8b 04 10 48 39 c8 74 34 48 29 d0 48 85 c0 48 89 87 30 01 00 00 74 30 <48> 8b 00 5d c3 48 8b 87 00 01 00 00 48 8d 97 00 01 00 00 48 39
Jun 12 02:32:12 storage-server kernel: [31051.692787] RIP  [<ffffffffa01d5d4b>] zio_walk_parents+0x3b/0x70 [zfs]
Jun 12 02:32:12 storage-server kernel: [31051.692929]  RSP <ffff8803fb155cf8>
Jun 12 02:32:12 storage-server kernel: [31051.700972] ---[ end trace b3d5c5433a80497f ]---
```

Another one

```
Jun 12 10:57:38 storage-server kernel: [19882.156737] general protection fault: 0000 [#1] SMP 
Jun 12 10:57:38 storage-server kernel: [19882.156770] Modules linked in: ib_iser rdma_cm iw_cm ib_cm ib_sa ib_mad ib_core ib_addr iscsi_tcp libiscsi_tcp libiscsi scsi_transport_iscsi snd_hda_codec_hdmi snd_hda_codec_realtek eeepc_wmi asus_wmi sparse_keymap mxm_wmi crc32_pclmul aesni_intel dm_multipath aes_x86_64 scsi_dh lrw gf128mul glue_helper ablk_helper ghash_clmulni_intel cryptd kvm coretemp crct10dif_pclmul intel_powerclamp x86_pkg_temp_thermal serio_raw snd_hda_intel mei_me i915 snd_hda_codec snd_hwdep snd_pcm snd_page_alloc drm_kms_helper mei drm snd_timer snd i2c_algo_bit soundcore shpchp video acpi_pad mac_hid wmi lp nfsd parport auth_rpcgss nfs_acl nfs lockd sunrpc fscache zfs(POX) zunicode(POX) zcommon(POX) znvpair(POX) spl(OX) zavl(POX) psmouse ahci r8169 libahci mii dm_mirror dm_region_hash dm_log
Jun 12 10:57:38 storage-server kernel: [19882.157058] CPU: 3 PID: 2870 Comm: z_rd_int/1 Tainted: P           OX 3.13.0-53-generic #89-Ubuntu
Jun 12 10:57:38 storage-server kernel: [19882.157080] Hardware name: ASUS All Series/Z97-K, BIOS 2604 05/20/2015
Jun 12 10:57:38 storage-server kernel: [19882.157097] task: ffff8803faef4800 ti: ffff8803fa8ca000 task.ti: ffff8803fa8ca000
Jun 12 10:57:38 storage-server kernel: [19882.157115] RIP: 0010:[<ffffffffa01d2b72>]  [<ffffffffa01d2b72>] zio_destroy+0x52/0xd0 [zfs]
Jun 12 10:57:38 storage-server kernel: [19882.157179] RSP: 0018:ffff8803fa8cbd68  EFLAGS: 00010246
Jun 12 10:57:38 storage-server kernel: [19882.157199] RAX: ffff88029d233e60 RBX: ffff88029d233d40 RCX: 00000001801d001a
Jun 12 10:57:38 storage-server kernel: [19882.157217] RDX: ffef88029d233e60 RSI: 00000000801d001a RDI: ffff88029d233d40
Jun 12 10:57:38 storage-server kernel: [19882.157235] RBP: ffff8803fa8cbd70 R08: 0000000000000000 R09: 0000000000000001
Jun 12 10:57:38 storage-server kernel: [19882.157252] R10: ffffffffa007a027 R11: ffffea000c069800 R12: ffff8803faef4800
Jun 12 10:57:38 storage-server kernel: [19882.157269] R13: 0000000000000000 R14: 0000000000200000 R15: 0000000000000000
Jun 12 10:57:38 storage-server kernel: [19882.157287] FS:  0000000000000000(0000) GS:ffff88041fb80000(0000) knlGS:0000000000000000
Jun 12 10:57:38 storage-server kernel: [19882.157307] CS:  0010 DS: 0000 ES: 0000 CR0: 0000000080050033
Jun 12 10:57:38 storage-server kernel: [19882.157321] CR2: 00007fdd202fea58 CR3: 0000000001c0e000 CR4: 00000000001407e0
Jun 12 10:57:38 storage-server kernel: [19882.157338] Stack:
Jun 12 10:57:38 storage-server kernel: [19882.157344]  ffff88029d233d40 ffff8803fa8cbde8 ffffffffa01d8a1d ffffffff81729642
Jun 12 10:57:38 storage-server kernel: [19882.157366]  ffff88029d233d40 ffff8803faef4800 ffff8803faef4800 ffff88029d234090
Jun 12 10:57:38 storage-server kernel: [19882.157388]  ffff88029d234068 ffff88029d233d40 ffff8803faef4800 0000000000200000
Jun 12 10:57:38 storage-server kernel: [19882.157409] Call Trace:
Jun 12 10:57:38 storage-server kernel: [19882.157453]  [<ffffffffa01d8a1d>] zio_done+0x9fd/0xe60 [zfs]
Jun 12 10:57:38 storage-server kernel: [19882.157470]  [<ffffffff81729642>] ? mutex_lock+0x12/0x2f
Jun 12 10:57:38 storage-server kernel: [19882.157516]  [<ffffffffa01d4228>] zio_execute+0xc8/0x180 [zfs]
Jun 12 10:57:38 storage-server kernel: [19882.157558]  [<ffffffffa007b200>] taskq_thread+0x1b0/0x360 [spl]
Jun 12 10:57:38 storage-server kernel: [19882.157602]  [<ffffffff8109ab20>] ? wake_up_state+0x20/0x20
Jun 12 10:57:38 storage-server kernel: [19882.157662]  [<ffffffffa007b050>] ? taskq_cancel_id+0x120/0x120 [spl]
Jun 12 10:57:38 storage-server kernel: [19882.157721]  [<ffffffff8108b6b2>] kthread+0xd2/0xf0
Jun 12 10:57:38 storage-server kernel: [19882.157755]  [<ffffffff8108b5e0>] ? kthread_create_on_node+0x1c0/0x1c0
Jun 12 10:57:38 storage-server kernel: [19882.157808]  [<ffffffff81733868>] ret_from_fork+0x58/0x90
Jun 12 10:57:38 storage-server kernel: [19882.157845]  [<ffffffff8108b5e0>] ? kthread_create_on_node+0x1c0/0x1c0
Jun 12 10:57:38 storage-server kernel: [19882.157892] Code: 00 00 00 ad de 48 89 87 00 01 00 00 48 b8 00 02 20 00 00 00 ad de 48 8b 97 20 01 00 00 48 89 87 08 01 00 00 48 8b 87 28 01 00 00 <48> 89 42 08 48 89 10 48 b8 00 01 10 00 00 00 ad de 48 89 87 20 
Jun 12 10:57:38 storage-server kernel: [19882.158133] RIP  [<ffffffffa01d2b72>] zio_destroy+0x52/0xd0 [zfs]
Jun 12 10:57:38 storage-server kernel: [19882.158241]  RSP <ffff8803fa8cbd68>
Jun 12 10:57:38 storage-server kernel: [19882.163224] ---[ end trace e89c9a3e79ee4b41 ]---
```

Another one

```
Jun 12 11:00:32 storage-server kernel: [20056.477150] general protection fault: 0000 [#2] SMP 
Jun 12 11:00:32 storage-server kernel: [20056.477218] Modules linked in: ib_iser rdma_cm iw_cm ib_cm ib_sa ib_mad ib_core ib_addr iscsi_tcp libiscsi_tcp libiscsi scsi_transport_iscsi snd_hda_codec_hdmi snd_hda_codec_realtek eeepc_wmi asus_wmi sparse_keymap mxm_wmi crc32_pclmul aesni_intel dm_multipath aes_x86_64 scsi_dh lrw gf128mul glue_helper ablk_helper ghash_clmulni_intel cryptd kvm coretemp crct10dif_pclmul intel_powerclamp x86_pkg_temp_thermal serio_raw snd_hda_intel mei_me i915 snd_hda_codec snd_hwdep snd_pcm snd_page_alloc drm_kms_helper mei drm snd_timer snd i2c_algo_bit soundcore shpchp video acpi_pad mac_hid wmi lp nfsd parport auth_rpcgss nfs_acl nfs lockd sunrpc fscache zfs(POX) zunicode(POX) zcommon(POX) znvpair(POX) spl(OX) zavl(POX) psmouse ahci r8169 libahci mii dm_mirror dm_region_hash dm_log
Jun 12 11:00:32 storage-server kernel: [20056.490355] CPU: 3 PID: 2871 Comm: z_rd_int/2 Tainted: P      D    OX 3.13.0-53-generic #89-Ubuntu
Jun 12 11:00:32 storage-server kernel: [20056.492333] Hardware name: ASUS All Series/Z97-K, BIOS 2604 05/20/2015
Jun 12 11:00:32 storage-server kernel: [20056.494539] task: ffff8803faef3000 ti: ffff8803fc9a8000 task.ti: ffff8803fc9a8000
Jun 12 11:00:32 storage-server kernel: [20056.496593] RIP: 0010:[<ffffffffa01d4e96>]  [<ffffffffa01d4e96>] zio_add_child+0xb6/0x170 [zfs]
Jun 12 11:00:32 storage-server kernel: [20056.498851] RSP: 0018:ffff8803fc9a9c90  EFLAGS: 00010282
Jun 12 11:00:32 storage-server kernel: [20056.501101] RAX: ffff8802e46affe0 RBX: ffff88025f3afa80 RCX: ffff8802e46affc0
Jun 12 11:00:32 storage-server kernel: [20056.503375] RDX: ffef88029d231b60 RSI: 0000000000004210 RDI: ffff88029d231d68
Jun 12 11:00:32 storage-server kernel: [20056.505616] RBP: ffff8803fc9a9cc0 R08: 00000000000165b0 R09: ffffffffa0078d19
Jun 12 11:00:32 storage-server kernel: [20056.507519] R10: ffff880405612868 R11: 000000000000135b R12: ffff88029d231a40
Jun 12 11:00:32 storage-server kernel: [20056.509318] R13: ffff8802e46affc0 R14: ffff88025f3afda8 R15: ffff88029d231d68
Jun 12 11:00:32 storage-server kernel: [20056.511109] FS:  0000000000000000(0000) GS:ffff88041fb80000(0000) knlGS:0000000000000000
Jun 12 11:00:32 storage-server kernel: [20056.512932] CS:  0010 DS: 0000 ES: 0000 CR0: 0000000080050033
Jun 12 11:00:32 storage-server kernel: [20056.514730] CR2: 00007f36fb31b000 CR3: 0000000001c0e000 CR4: 00000000001407e0
Jun 12 11:00:32 storage-server kernel: [20056.516523] Stack:
Jun 12 11:00:32 storage-server kernel: [20056.518312]  ffff8803faef3000 ffff88029d231a40 ffff88029d231a40 ffff8803ff172508
Jun 12 11:00:32 storage-server kernel: [20056.520141]  ffff88025f3afa80 0000000000000000 ffff8803fc9a9d60 ffffffffa0199d89
Jun 12 11:00:32 storage-server kernel: [20056.522010]  0000000000040800 ffffffffa0199690 0000000000000000 ffff880409557700
Jun 12 11:00:32 storage-server kernel: [20056.523847] Call Trace:
Jun 12 11:00:32 storage-server kernel: [20056.525718]  [<ffffffffa0199d89>] vdev_queue_io_to_issue+0x5c9/0x860 [zfs]
Jun 12 11:00:32 storage-server kernel: [20056.527620]  [<ffffffffa0199690>] ? vdev_queue_timestamp_compare+0x40/0x40 [zfs]
Jun 12 11:00:32 storage-server kernel: [20056.529500]  [<ffffffffa019a525>] vdev_queue_io_done+0x195/0x270 [zfs]
Jun 12 11:00:32 storage-server kernel: [20056.531420]  [<ffffffffa01d2f08>] zio_vdev_io_done+0x88/0x180 [zfs]
Jun 12 11:00:32 storage-server kernel: [20056.533294]  [<ffffffffa01d4228>] zio_execute+0xc8/0x180 [zfs]
Jun 12 11:00:32 storage-server kernel: [20056.535162]  [<ffffffffa007b200>] taskq_thread+0x1b0/0x360 [spl]
Jun 12 11:00:32 storage-server kernel: [20056.536978]  [<ffffffff8109ab20>] ? wake_up_state+0x20/0x20
Jun 12 11:00:32 storage-server kernel: [20056.538811]  [<ffffffffa007b050>] ? taskq_cancel_id+0x120/0x120 [spl]
Jun 12 11:00:32 storage-server kernel: [20056.540620]  [<ffffffff8108b6b2>] kthread+0xd2/0xf0
Jun 12 11:00:32 storage-server kernel: [20056.542438]  [<ffffffff8108b5e0>] ? kthread_create_on_node+0x1c0/0x1c0
Jun 12 11:00:32 storage-server kernel: [20056.544249]  [<ffffffff81733868>] ret_from_fork+0x58/0x90
Jun 12 11:00:32 storage-server kernel: [20056.546070]  [<ffffffff8108b5e0>] ? kthread_create_on_node+0x1c0/0x1c0
Jun 12 11:00:32 storage-server kernel: [20056.547902] Code: 31 d2 8b 43 40 48 c1 e0 04 80 7b 4e 00 0f 94 c2 49 01 94 04 b0 02 00 00 4c 89 e8 49 03 84 24 18 01 00 00 49 8b 94 24 20 01 00 00 <48> 89 42 08 48 89 10 49 8d 94 24 20 01 00 00 48 89 50 08 49 89 
Jun 12 11:00:32 storage-server kernel: [20056.551878] RIP  [<ffffffffa01d4e96>] zio_add_child+0xb6/0x170 [zfs]
Jun 12 11:00:32 storage-server kernel: [20056.553854]  RSP <ffff8803fc9a9c90>
Jun 12 11:00:32 storage-server kernel: [20056.555776] ---[ end trace e89c9a3e79ee4b42 ]---
```

**Comment by behlendorf** [comment]
Thanks for filing this so we can track it.  Unfortunately I don't have a quick fix for you, it's not clear why your hitting the GPF when others aren't.  Is this a new issue your seeing just with master or can you reproduce this wit 0.6.4.2?

**Comment by JustInVTime** [comment]
Hello all,

I'm also facing a 'frozen' file server, during high IO load (making a backup). Which, in our case, resulted in the following kernel panic:

```
Jun 17 11:57:02 freddie kernel: [12977.415227] BUG: unable to handle kernel NULL pointer dereference at 0000000000000060
Jun 17 11:57:02 freddie kernel: [12977.415258] IP: [<ffffffffa02889ae>] sa_find_idx_tab+0x5e/0x1e0 [zfs]
Jun 17 11:57:02 freddie kernel: [12977.415300] PGD 44bc13067 PUD 44bec9067 PMD 0 
Jun 17 11:57:02 freddie kernel: [12977.415317] Oops: 0000 [#1] SMP 
Jun 17 11:57:02 freddie kernel: [12977.415329] Modules linked in: vhost_net vhost macvtap macvlan ip6table_filter ip6_tables iptable_filter ip_tables ebtable_nat ebtables x_tables bridge gpio_ich intel_rapl x86_pkg_temp_thermal intel_powerclamp coretemp kvm_intel kvm serio_raw sb_edac edac_core snd_hda_codec_hdmi lpc_ich snd_hda_codec_realtek mei_me snd_hda_intel mei snd_hda_codec snd_hwdep snd_pcm snd_page_alloc snd_timer ioatdma 8021q snd garp stp dca mrp llc soundcore lp parport mac_hid shpchp nfsd auth_rpcgss nfs_acl nfs lockd sunrpc fscache zfs(POX) zunicode(POX) zcommon(POX) znvpair(POX) spl(OX) zavl(POX) nls_iso8859_1 dm_crypt osst st hid_apple usbhid hid usb_storage mxm_wmi crct10dif_pclmul crc32_pclmul ghash_clmulni_intel aesni_intel aes_x86_64 lrw gf128mul glue_helper ablk_helper e1000e cryptd firewire_ohci mptspi psmouse mptscsih firewire_core ahci ptp mptbase crc_itu_t libahci pps_core wmi
Jun 17 11:57:02 freddie kernel: [12977.415637] CPU: 3 PID: 4199 Comm: bacula-fd Tainted: P           OX 3.13.0-55-generic #92-Ubuntu
Jun 17 11:57:02 freddie kernel: [12977.415660] Hardware name: Gigabyte Technology Co., Ltd. To be filled by O.E.M./X79-UD5, BIOS F12 07/05/2012
Jun 17 11:57:02 freddie kernel: [12977.415685] task: ffff88045ec13000 ti: ffff88044a860000 task.ti: ffff88044a860000
Jun 17 11:57:02 freddie kernel: [12977.415704] RIP: 0010:[<ffffffffa02889ae>]  [<ffffffffa02889ae>] sa_find_idx_tab+0x5e/0x1e0 [zfs]
Jun 17 11:57:02 freddie kernel: [12977.415741] RSP: 0018:ffff88044a8616d8  EFLAGS: 00010246
Jun 17 11:57:02 freddie kernel: [12977.415755] RAX: 0000000000000000 RBX: ffff8801cdf55200 RCX: ffff8800c82e1401
Jun 17 11:57:02 freddie kernel: [12977.415773] RDX: 0000000000000000 RSI: ffff8800c82e1400 RDI: ffff88044a8616f0
Jun 17 11:57:02 freddie kernel: [12977.415791] RBP: ffff88044a861790 R08: 00000000000175c0 R09: ffffffffa01bcd19
Jun 17 11:57:02 freddie kernel: [12977.415809] R10: ffff8807fea66868 R11: ffffea0013a36a00 R12: ffff8807fe0b5100
Jun 17 11:57:02 freddie kernel: [12977.415827] R13: 0000000000000001 R14: 0000000000000060 R15: 0000000000000000
Jun 17 11:57:02 freddie kernel: [12977.415845] FS:  00007fa04bf21700(0000) GS:ffff88082fcc0000(0000) knlGS:0000000000000000
Jun 17 11:57:02 freddie kernel: [12977.415865] CS:  0010 DS: 0000 ES: 0000 CR0: 0000000080050033
Jun 17 11:57:02 freddie kernel: [12977.415880] CR2: 0000000000000060 CR3: 000000044c6a0000 CR4: 00000000000427e0
Jun 17 11:57:02 freddie kernel: [12977.415898] Stack:
Jun 17 11:57:02 freddie kernel: [12977.415904]  0000002c4a8616f0 ffff8807fd798000 ffff8800c82e1401 ffffffffa01bcd19
Jun 17 11:57:02 freddie kernel: [12977.415926]  0000000000000000 0000000000000000 ffff88044a861720 ffffffffa0288680
Jun 17 11:57:02 freddie kernel: [12977.415949]  ffff8807fea66000 0000000000000102 ffffffffa01bcdb6 ffff880780a535a8
Jun 17 11:57:02 freddie kernel: [12977.415971] Call Trace:
Jun 17 11:57:02 freddie kernel: [12977.415983]  [<ffffffffa01bcd19>] ? spl_kmem_cache_alloc+0xb9/0x790 [spl]
Jun 17 11:57:02 freddie kernel: [12977.416014]  [<ffffffffa0288680>] ? sa_cache_constructor+0x50/0x70 [zfs]
Jun 17 11:57:02 freddie kernel: [12977.416034]  [<ffffffffa01bcdb6>] ? spl_kmem_cache_alloc+0x156/0x790 [spl]
Jun 17 11:57:02 freddie kernel: [12977.416064]  [<ffffffffa028a71b>] sa_build_index+0x6b/0x140 [zfs]
Jun 17 11:57:02 freddie kernel: [12977.416093]  [<ffffffffa028c008>] sa_handle_get_from_db+0xb8/0x100 [zfs]
Jun 17 11:57:02 freddie kernel: [12977.416128]  [<ffffffffa02de087>] zfs_znode_sa_init.isra.8+0xb7/0xf0 [zfs]
Jun 17 11:57:02 freddie kernel: [12977.416162]  [<ffffffffa02def17>] zfs_znode_alloc+0xe7/0x590 [zfs]
Jun 17 11:57:02 freddie kernel: [12977.416195]  [<ffffffffa02ed490>] ? zio_done+0x700/0xe90 [zfs]
Jun 17 11:57:02 freddie kernel: [12977.416213]  [<ffffffffa01be027>] ? spl_kmem_cache_free+0x167/0x1f0 [spl]
Jun 17 11:57:02 freddie kernel: [12977.416246]  [<ffffffffa02e7c7d>] ? zio_destroy+0xcd/0xd0 [zfs]
Jun 17 11:57:02 freddie kernel: [12977.416277]  [<ffffffffa02eb1da>] ? zio_wait+0x16a/0x210 [zfs]
Jun 17 11:57:02 freddie kernel: [12977.416301]  [<ffffffffa024b757>] ? dbuf_read+0x2d7/0xa10 [zfs]
Jun 17 11:57:02 freddie kernel: [12977.416325]  [<ffffffffa024a766>] ? dbuf_rele+0x36/0x40 [zfs]
Jun 17 11:57:02 freddie kernel: [12977.416350]  [<ffffffffa0255e8b>] ? dmu_object_info_from_dnode+0x10b/0x120 [zfs]
Jun 17 11:57:02 freddie kernel: [12977.416384]  [<ffffffffa02e18e1>] zfs_zget+0x291/0x310 [zfs]
Jun 17 11:57:02 freddie kernel: [12977.416415]  [<ffffffffa02c05b4>] zfs_dirent_lock+0x474/0x5b0 [zfs]
Jun 17 11:57:02 freddie kernel: [12977.416446]  [<ffffffffa02c0771>] zfs_dirlook+0x81/0x370 [zfs]
Jun 17 11:57:02 freddie kernel: [12977.416477]  [<ffffffffa02d5785>] zfs_lookup+0x2d5/0x320 [zfs]
Jun 17 11:57:02 freddie kernel: [12977.416507]  [<ffffffffa02f218e>] zpl_lookup+0x8e/0x110 [zfs]
Jun 17 11:57:02 freddie kernel: [12977.416524]  [<ffffffff811c7e9d>] lookup_real+0x1d/0x50
Jun 17 11:57:02 freddie kernel: [12977.416539]  [<ffffffff811c87d3>] __lookup_hash+0x33/0x40
Jun 17 11:57:02 freddie kernel: [12977.416555]  [<ffffffff8171fda7>] lookup_slow+0x42/0xa7
Jun 17 11:57:02 freddie kernel: [12977.416569]  [<ffffffff811cbd45>] path_lookupat+0x725/0x790
Jun 17 11:57:02 freddie kernel: [12977.416585]  [<ffffffff811a3265>] ? kmem_cache_alloc+0x35/0x1e0
Jun 17 11:57:02 freddie kernel: [12977.416601]  [<ffffffff811ce13f>] ? getname_flags+0x4f/0x190
Jun 17 11:57:02 freddie kernel: [12977.416617]  [<ffffffff811cbddb>] filename_lookup+0x2b/0xc0
Jun 17 11:57:02 freddie kernel: [12977.416632]  [<ffffffff811cee14>] user_path_at_empty+0x54/0x90
Jun 17 11:57:02 freddie kernel: [12977.416648]  [<ffffffff811cee61>] user_path_at+0x11/0x20
Jun 17 11:57:02 freddie kernel: [12977.416663]  [<ffffffff811c2fc0>] vfs_fstatat+0x50/0xa0
Jun 17 11:57:02 freddie kernel: [12977.416677]  [<ffffffff811c34a2>] SYSC_newlstat+0x22/0x40
Jun 17 11:57:02 freddie kernel: [12977.416691]  [<ffffffff811d1c8d>] ? SyS_getdents+0xed/0x120
Jun 17 11:57:02 freddie kernel: [12977.416706]  [<ffffffff811d18f0>] ? fillonedir+0xe0/0xe0
Jun 17 11:57:02 freddie kernel: [12977.416721]  [<ffffffff811c36be>] SyS_newlstat+0xe/0x10
Jun 17 11:57:02 freddie kernel: [12977.417634]  [<ffffffff81733d5d>] system_call_fastpath+0x1a/0x1f
Jun 17 11:57:02 freddie kernel: [12977.418551] Code: 00 00 0f 84 7e 01 00 00 49 8d 7c 24 70 48 8d 95 58 ff ff ff 48 8d b5 60 ff ff ff 48 89 45 90 e8 89 f7 df ff 4c 8d 70 60 49 89 c7 <48> 8b 40 60 49 39 c6 0f 84 47 01 00 00 49 2b 47 58 48 85 c0 74 
Jun 17 11:57:02 freddie kernel: [12977.420605] RIP  [<ffffffffa02889ae>] sa_find_idx_tab+0x5e/0x1e0 [zfs]
Jun 17 11:57:02 freddie kernel: [12977.421601]  RSP <ffff88044a8616d8>
Jun 17 11:57:02 freddie kernel: [12977.422578] CR2: 0000000000000060
Jun 17 11:57:02 freddie kernel: [12977.425621] ---[ end trace 8b31f492badc059f ]---
```

Some system information, please let me know if you guys need more info.
```
uname -a:
Linux freddie 3.13.0-55-generic #92-Ubuntu SMP Sun Jun 14 18:32:20 UTC 2015 x86_64 x86_64 x86_64 GNU/Linux

lsb_release -a:
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 14.04.2 LTS
Release:	14.04
Codename:	trusty

We use the stable repository, dpkg --list | grep zfs:
ii  dkms                                 2.2.0.3-1.1ubuntu5.14.04.1+zfs10~trusty all          Dynamic Kernel Module Support Framework
ii  libzfs2                              0.6.4.1-1~trusty                        amd64        Native OpenZFS filesystem library for Linux
ii  mountall                             2.53-zfs1                               amd64        filesystem mounting tool
ii  ubuntu-zfs                           8~trusty                                amd64        Native ZFS filesystem metapackage for Ubuntu.
ii  zfs-dkms                             0.6.4.1-1~trusty                        amd64        Native OpenZFS filesystem kernel modules for Linux
ii  zfs-doc                              0.6.4.1-1~trusty                        amd64        Native OpenZFS filesystem documentation and examples.
ii  zfsutils                             0.6.4.1-1~trusty                        amd64        Native OpenZFS management utilities for Linux

```

Could this bug be related? Else I will create a different bug report if desired. Please let me know how to proceed to help you guys squash this bug/issue.

**Comment by F0x06** [comment]
@behlendorf 
```
Thanks for filing this so we can track it. Unfortunately I don't have a quick fix for you, it's not clear why your hitting the GPF when others aren't. Is this a new issue your seeing just with master or can you reproduce this wit 0.6.4.2?
```

Yes, same problem with version 0.6.4.1 (release), for that reason i tried master branch, but the problem persists.

Last day i tried different settings to investigate aboute this bug, i reduced zfs freeze frequency by limiting ARC maximum memory to 5.0 GB (i have 16), now the server can support huge io for max 2 days (before it was 1-2h) after it hangs again...

This is the four last crash logs (all are from the same crash) (a bit different):

```
Jun 16 10:59:49 server1 kernel: [79278.675616] general protection fault: 0000 [#13] SMP 
Jun 16 10:59:49 server1 kernel: [79278.675677] Modules linked in: ib_iser rdma_cm iw_cm ib_cm ib_sa ib_mad ib_core ib_addr iscsi_tcp libiscsi_tcp libiscsi scsi_transport_iscsi snd_hda_codec_hdmi crc32_pclmul aesni_intel aes_x86_64 lrw gf128mul glue_helper ablk_helper ghash_clmulni_intel cryptd i915 eeepc_wmi asus_wmi sparse_keymap nfsd snd_hda_codec_realtek auth_rpcgss kvm snd_hda_codec_generic nfs_acl mxm_wmi nfs snd_hda_intel coretemp drm_kms_helper crct10dif_pclmul snd_hda_controller drm intel_powerclamp lockd intel_rapl snd_hda_codec iosf_mbi dm_multipath snd_hwdep x86_pkg_temp_thermal scsi_dh grace snd_pcm snd_timer serio_raw lpc_ich mei_me snd shpchp soundcore i2c_algo_bit mei sunrpc 8250_fintek wmi tpm_infineon lp mac_hid video acpi_pad fscache parport zfs(POE) zunicode(POE) zcommon(POE) znvpair(POE) spl(OE) zavl(POE) psmouse ahci r8169 libahci mii dm_mirror dm_region_hash dm_log
Jun 16 10:59:49 server1 kernel: [79278.675689] CPU: 1 PID: 2832 Comm: zfs Tainted: P      D    OE  3.19.0-031900-generic #201504091832
Jun 16 10:59:49 server1 kernel: [79278.675691] Hardware name: ASUS All Series/Z97-K, BIOS 2604 05/20/2015
Jun 16 10:59:49 server1 kernel: [79278.675694] task: ffff8800d3875850 ti: ffff8803fe8dc000 task.ti: ffff8803fe8dc000
Jun 16 10:59:49 server1 kernel: [79278.675706] RIP: 0010:[<ffffffff811d7278>]  [<ffffffff811d7278>] kmem_cache_alloc+0x78/0x210
Jun 16 10:59:49 server1 kernel: [79278.675708] RSP: 0018:ffff8803fe8df8e8  EFLAGS: 00010286
Jun 16 10:59:49 server1 kernel: [79278.675710] RAX: 0000000000000000 RBX: 0000000000004210 RCX: 00000000060ba345
Jun 16 10:59:49 server1 kernel: [79278.675712] RDX: 00000000060ba344 RSI: 0000000000004210 RDI: 000000000001dbd0
Jun 16 10:59:49 server1 kernel: [79278.675713] RBP: ffff8803fe8df938 R08: ffff88041fa9dbd0 R09: 0000000000000000
Jun 16 10:59:49 server1 kernel: [79278.675715] R10: ffff880036831868 R11: 0000000000000000 R12: ffff8800d357a700
Jun 16 10:59:49 server1 kernel: [79278.675716] R13: ffef88029dc16040 R14: ffff8800d357a700 R15: ffffffffc01bbd59
Jun 16 10:59:49 server1 kernel: [79278.675719] FS:  00007fd220307bc0(0000) GS:ffff88041fa80000(0000) knlGS:0000000000000000
Jun 16 10:59:49 server1 kernel: [79278.675721] CS:  0010 DS: 0000 ES: 0000 CR0: 0000000080050033
Jun 16 10:59:49 server1 kernel: [79278.675723] CR2: 00000000025c0000 CR3: 0000000406739000 CR4: 00000000001407e0
Jun 16 10:59:49 server1 kernel: [79278.675724] Stack:
Jun 16 10:59:49 server1 kernel: [79278.675728]  ffff8803fe8df908 ffffffffc0272983 ffff88040a7ed500 000042100a976000
Jun 16 10:59:49 server1 kernel: [79278.675731]  ffff8803fe8df9c8 0000000000000000 0000000000000000 0000000000000000
Jun 16 10:59:49 server1 kernel: [79278.675735]  ffff8800d357a700 ffff8800d3875850 ffff8803fe8df9f8 ffffffffc01bbd59
Jun 16 10:59:49 server1 kernel: [79278.675736] Call Trace:
Jun 16 10:59:49 server1 kernel: [79278.675783]  [<ffffffffc0272983>] ? dbuf_cons+0xa3/0x110 [zfs]
Jun 16 10:59:49 server1 kernel: [79278.675801]  [<ffffffffc01bbd59>] spl_kmem_cache_alloc+0xb9/0x7c0 [spl]
Jun 16 10:59:49 server1 kernel: [79278.675808]  [<ffffffff817d0be6>] ? mutex_lock+0x16/0x37
Jun 16 10:59:49 server1 kernel: [79278.675847]  [<ffffffffc0273633>] ? dbuf_rele_and_unlock+0x2d3/0x450 [zfs]
Jun 16 10:59:49 server1 kernel: [79278.675936]  [<ffffffffc03182dd>] zio_create+0x3d/0x500 [zfs]
Jun 16 10:59:49 server1 kernel: [79278.676018]  [<ffffffffc03189d1>] zio_null+0x61/0x70 [zfs]
Jun 16 10:59:49 server1 kernel: [79278.676098]  [<ffffffffc03189fe>] zio_root+0x1e/0x20 [zfs]
Jun 16 10:59:49 server1 kernel: [79278.676135]  [<ffffffffc0275686>] dbuf_read+0x626/0x950 [zfs]
Jun 16 10:59:49 server1 kernel: [79278.676184]  [<ffffffffc028fe6f>] ? dnode_rele_and_unlock+0x5f/0xa0 [zfs]
Jun 16 10:59:49 server1 kernel: [79278.676231]  [<ffffffffc028fee9>] ? dnode_rele+0x39/0x40 [zfs]
Jun 16 10:59:49 server1 kernel: [79278.676271]  [<ffffffffc027e92d>] ? dmu_buf_hold_noread+0x8d/0x130 [zfs]
Jun 16 10:59:49 server1 kernel: [79278.676312]  [<ffffffffc027ea20>] dmu_buf_hold+0x50/0x80 [zfs]
Jun 16 10:59:49 server1 kernel: [79278.676389]  [<ffffffffc02e5491>] zap_lockdir+0x61/0x8f0 [zfs]
Jun 16 10:59:49 server1 kernel: [79278.676424]  [<ffffffffc027570f>] ? dbuf_read+0x6af/0x950 [zfs]
Jun 16 10:59:49 server1 kernel: [79278.676470]  [<ffffffffc028faf2>] ? dnode_hold_impl+0x402/0x600 [zfs]
Jun 16 10:59:49 server1 kernel: [79278.676515]  [<ffffffffc028fe6f>] ? dnode_rele_and_unlock+0x5f/0xa0 [zfs]
Jun 16 10:59:49 server1 kernel: [79278.676592]  [<ffffffffc02e616f>] zap_lookup_norm+0x3f/0x1c0 [zfs]
Jun 16 10:59:49 server1 kernel: [79278.676667]  [<ffffffffc02e6323>] zap_lookup+0x33/0x40 [zfs]
Jun 16 10:59:49 server1 kernel: [79278.676722]  [<ffffffffc02a2316>] dsl_dir_hold+0x156/0x220 [zfs]
Jun 16 10:59:49 server1 kernel: [79278.676773]  [<ffffffffc0296881>] dsl_dataset_hold+0x31/0x1f0 [zfs]
Jun 16 10:59:49 server1 kernel: [79278.676826]  [<ffffffffc02a54fb>] ? dsl_pool_hold+0x4b/0x60 [zfs]
Jun 16 10:59:49 server1 kernel: [79278.676869]  [<ffffffffc0281453>] dmu_objset_hold+0x53/0xb0 [zfs]
Jun 16 10:59:49 server1 kernel: [79278.676876]  [<ffffffff811d8331>] ? __kmalloc+0x1d1/0x260
Jun 16 10:59:49 server1 kernel: [79278.676890]  [<ffffffffc01ba91c>] ? strdup+0x3c/0x60 [spl]
Jun 16 10:59:49 server1 kernel: [79278.676965]  [<ffffffffc02f46a1>] zfs_ioc_objset_stats+0x21/0x50 [zfs]
Jun 16 10:59:49 server1 kernel: [79278.676978]  [<ffffffffc01ba932>] ? strdup+0x52/0x60 [spl]
Jun 16 10:59:49 server1 kernel: [79278.677052]  [<ffffffffc02f7fed>] zfsdev_ioctl+0x49d/0x4e0 [zfs]
Jun 16 10:59:49 server1 kernel: [79278.677059]  [<ffffffff81207635>] do_vfs_ioctl+0x75/0x320
Jun 16 10:59:49 server1 kernel: [79278.677064]  [<ffffffff81207971>] SyS_ioctl+0x91/0xb0
Jun 16 10:59:49 server1 kernel: [79278.677072]  [<ffffffff817d2d6d>] system_call_fastpath+0x16/0x1b
Jun 16 10:59:49 server1 kernel: [79278.677105] Code: 7e 49 8b 50 08 4d 8b 28 49 8b 40 10 4d 85 ed 0f 84 76 01 00 00 48 85 c0 0f 84 6d 01 00 00 49 63 44 24 20 49 8b 3c 24 48 8d 4a 01 <49> 8b 5c 05 00 4c 89 e8 65 48 0f c7 0f 0f 94 c0 84 c0 74 b6 49 
Jun 16 10:59:49 server1 kernel: [79278.677112] RIP  [<ffffffff811d7278>] kmem_cache_alloc+0x78/0x210
Jun 16 10:59:49 server1 kernel: [79278.677113]  RSP <ffff8803fe8df8e8>
Jun 16 10:59:49 server1 kernel: [79278.677117] ---[ end trace 3c48fcfab6da8eab ]---
```

```
Jun 16 10:58:09 server1 kernel: [79178.949808] general protection fault: 0000 [#11] SMP 
Jun 16 10:58:09 server1 kernel: [79178.949868] Modules linked in: ib_iser rdma_cm iw_cm ib_cm ib_sa ib_mad ib_core ib_addr iscsi_tcp libiscsi_tcp libiscsi scsi_transport_iscsi snd_hda_codec_hdmi crc32_pclmul aesni_intel aes_x86_64 lrw gf128mul glue_helper ablk_helper ghash_clmulni_intel cryptd i915 eeepc_wmi asus_wmi sparse_keymap nfsd snd_hda_codec_realtek auth_rpcgss kvm snd_hda_codec_generic nfs_acl mxm_wmi nfs snd_hda_intel coretemp drm_kms_helper crct10dif_pclmul snd_hda_controller drm intel_powerclamp lockd intel_rapl snd_hda_codec iosf_mbi dm_multipath snd_hwdep x86_pkg_temp_thermal scsi_dh grace snd_pcm snd_timer serio_raw lpc_ich mei_me snd shpchp soundcore i2c_algo_bit mei sunrpc 8250_fintek wmi tpm_infineon lp mac_hid video acpi_pad fscache parport zfs(POE) zunicode(POE) zcommon(POE) znvpair(POE) spl(OE) zavl(POE) psmouse ahci r8169 libahci mii dm_mirror dm_region_hash dm_log
Jun 16 10:58:09 server1 kernel: [79178.949879] CPU: 1 PID: 1716 Comm: nfsd Tainted: P      D    OE  3.19.0-031900-generic #201504091832
Jun 16 10:58:09 server1 kernel: [79178.949881] Hardware name: ASUS All Series/Z97-K, BIOS 2604 05/20/2015
Jun 16 10:58:09 server1 kernel: [79178.949884] task: ffff8800d383e220 ti: ffff8803fedc0000 task.ti: ffff8803fedc0000
Jun 16 10:58:09 server1 kernel: [79178.949897] RIP: 0010:[<ffffffff811d7278>]  [<ffffffff811d7278>] kmem_cache_alloc+0x78/0x210
Jun 16 10:58:09 server1 kernel: [79178.949899] RSP: 0018:ffff8803fedc33f8  EFLAGS: 00010286
Jun 16 10:58:09 server1 kernel: [79178.949901] RAX: 0000000000000000 RBX: 0000000000004210 RCX: 00000000060ba345
Jun 16 10:58:09 server1 kernel: [79178.949903] RDX: 00000000060ba344 RSI: 0000000000004210 RDI: 000000000001dbd0
Jun 16 10:58:09 server1 kernel: [79178.949904] RBP: ffff8803fedc3448 R08: ffff88041fa9dbd0 R09: 0000000000000000
Jun 16 10:58:09 server1 kernel: [79178.949906] R10: ffff880036831868 R11: 0000000000000000 R12: ffff8800d357a700
Jun 16 10:58:09 server1 kernel: [79178.949907] R13: ffef88029dc16040 R14: ffff8800d357a700 R15: ffffffffc01bbd59
Jun 16 10:58:09 server1 kernel: [79178.949910] FS:  0000000000000000(0000) GS:ffff88041fa80000(0000) knlGS:0000000000000000
Jun 16 10:58:09 server1 kernel: [79178.949912] CS:  0010 DS: 0000 ES: 0000 CR0: 0000000080050033
Jun 16 10:58:09 server1 kernel: [79178.949914] CR2: 00007fdb0fa8c000 CR3: 0000000001c15000 CR4: 00000000001407e0
Jun 16 10:58:09 server1 kernel: [79178.949915] Stack:
Jun 16 10:58:09 server1 kernel: [79178.949919]  ffff8803fedc3458 ffffffff811d73eb 00000000000000d6 0000421000000128
Jun 16 10:58:09 server1 kernel: [79178.949923]  ffff8803fedc3438 0000000000000000 0000000000000000 0000000000000000
Jun 16 10:58:09 server1 kernel: [79178.949926]  ffff8800d357a700 ffff8800d383e220 ffff8803fedc3508 ffffffffc01bbd59
Jun 16 10:58:09 server1 kernel: [79178.949927] Call Trace:
Jun 16 10:58:09 server1 kernel: [79178.949935]  [<ffffffff811d73eb>] ? kmem_cache_alloc+0x1eb/0x210
Jun 16 10:58:09 server1 kernel: [79178.949955]  [<ffffffffc01bbd59>] spl_kmem_cache_alloc+0xb9/0x7c0 [spl]
Jun 16 10:58:09 server1 kernel: [79178.949970]  [<ffffffffc01bbdf5>] ? spl_kmem_cache_alloc+0x155/0x7c0 [spl]
Jun 16 10:58:09 server1 kernel: [79178.950069]  [<ffffffffc03182dd>] zio_create+0x3d/0x500 [zfs]
Jun 16 10:58:09 server1 kernel: [79178.950159]  [<ffffffffc03189d1>] zio_null+0x61/0x70 [zfs]
Jun 16 10:58:09 server1 kernel: [79178.950242]  [<ffffffffc03189fe>] zio_root+0x1e/0x20 [zfs]
Jun 16 10:58:09 server1 kernel: [79178.950281]  [<ffffffffc0275686>] dbuf_read+0x626/0x950 [zfs]
Jun 16 10:58:09 server1 kernel: [79178.950334]  [<ffffffffc028bb7a>] dmu_tx_check_ioerr+0x6a/0xf0 [zfs]
Jun 16 10:58:09 server1 kernel: [79178.950381]  [<ffffffffc028c035>] dmu_tx_count_write+0x435/0x6a0 [zfs]
Jun 16 10:58:09 server1 kernel: [79178.950420]  [<ffffffffc0273906>] ? dbuf_rele+0x36/0x40 [zfs]
Jun 16 10:58:09 server1 kernel: [79178.950435]  [<ffffffffc01bac40>] ? spl_kmem_zalloc+0xf0/0x1b0 [spl]
Jun 16 10:58:09 server1 kernel: [79178.950448]  [<ffffffffc01bac40>] ? spl_kmem_zalloc+0xf0/0x1b0 [spl]
Jun 16 10:58:09 server1 kernel: [79178.950460]  [<ffffffffc01bac40>] ? spl_kmem_zalloc+0xf0/0x1b0 [spl]
Jun 16 10:58:09 server1 kernel: [79178.950506]  [<ffffffffc028c2da>] dmu_tx_hold_write+0x3a/0x50 [zfs]
Jun 16 10:58:09 server1 kernel: [79178.950587]  [<ffffffffc0308e8c>] zfs_write+0x3cc/0xc20 [zfs]
Jun 16 10:58:09 server1 kernel: [79178.950599]  [<ffffffff817d0be6>] ? mutex_lock+0x16/0x37
Jun 16 10:58:09 server1 kernel: [79178.950680]  [<ffffffffc030cf14>] ? zfs_inode_update+0x184/0x190 [zfs]
Jun 16 10:58:09 server1 kernel: [79178.950688]  [<ffffffff811d80f1>] ? __kmalloc_node+0x231/0x2a0
Jun 16 10:58:09 server1 kernel: [79178.950702]  [<ffffffffc01baa87>] ? spl_kmem_alloc+0xf7/0x1c0 [spl]
Jun 16 10:58:09 server1 kernel: [79178.950715]  [<ffffffffc01baa87>] ? spl_kmem_alloc+0xf7/0x1c0 [spl]
Jun 16 10:58:09 server1 kernel: [79178.950796]  [<ffffffffc031f735>] zpl_aio_write+0xe5/0x150 [zfs]
Jun 16 10:58:09 server1 kernel: [79178.950802]  [<ffffffff811f4158>] do_sync_readv_writev+0x48/0x80
Jun 16 10:58:09 server1 kernel: [79178.950807]  [<ffffffff811f58d2>] do_readv_writev+0x1e2/0x2e0
Jun 16 10:58:09 server1 kernel: [79178.950888]  [<ffffffffc0303066>] ? zfs_open+0x86/0x120 [zfs]
Jun 16 10:58:09 server1 kernel: [79178.950968]  [<ffffffffc031f650>] ? zpl_compat_ioctl+0x10/0x10 [zfs]
Jun 16 10:58:09 server1 kernel: [79178.951046]  [<ffffffffc031fb70>] ? zpl_read+0xf0/0xf0 [zfs]
Jun 16 10:58:09 server1 kernel: [79178.951121]  [<ffffffffc031f220>] ? zpl_release+0x90/0x90 [zfs]
Jun 16 10:58:09 server1 kernel: [79178.951128]  [<ffffffff811f1fca>] ? do_dentry_open+0x29a/0x360
Jun 16 10:58:09 server1 kernel: [79178.951133]  [<ffffffff811f5a0b>] vfs_writev+0x3b/0x40
Jun 16 10:58:09 server1 kernel: [79178.951148]  [<ffffffffc085d4ff>] nfsd_vfs_write.isra.13+0xaf/0x240 [nfsd]
Jun 16 10:58:09 server1 kernel: [79178.951162]  [<ffffffffc085f7fc>] nfsd_write+0xec/0x100 [nfsd]
Jun 16 10:58:09 server1 kernel: [79178.951170]  [<ffffffff813b1281>] ? csum_partial+0x11/0x20
Jun 16 10:58:09 server1 kernel: [79178.951185]  [<ffffffffc0867631>] nfsd3_proc_write+0xb1/0x140 [nfsd]
Jun 16 10:58:09 server1 kernel: [79178.951198]  [<ffffffffc0858e55>] nfsd_dispatch+0xe5/0x230 [nfsd]
Jun 16 10:58:09 server1 kernel: [79178.951230]  [<ffffffffc054e9d2>] ? svc_tcp_adjust_wspace+0x12/0x30 [sunrpc]
Jun 16 10:58:09 server1 kernel: [79178.951258]  [<ffffffffc054d764>] svc_process_common+0x324/0x680 [sunrpc]
Jun 16 10:58:09 server1 kernel: [79178.951284]  [<ffffffffc054de23>] svc_process+0x113/0x1d0 [sunrpc]
Jun 16 10:58:09 server1 kernel: [79178.951295]  [<ffffffffc08589c7>] nfsd+0x117/0x190 [nfsd]
Jun 16 10:58:09 server1 kernel: [79178.951306]  [<ffffffffc08588b0>] ? nfsd_destroy+0x80/0x80 [nfsd]
Jun 16 10:58:09 server1 kernel: [79178.951314]  [<ffffffff81095d59>] kthread+0xc9/0xe0
Jun 16 10:58:09 server1 kernel: [79178.951319]  [<ffffffff81095c90>] ? flush_kthread_worker+0x90/0x90
Jun 16 10:58:09 server1 kernel: [79178.951327]  [<ffffffff817d2cbc>] ret_from_fork+0x7c/0xb0
Jun 16 10:58:09 server1 kernel: [79178.951331]  [<ffffffff81095c90>] ? flush_kthread_worker+0x90/0x90
Jun 16 10:58:09 server1 kernel: [79178.951365] Code: 7e 49 8b 50 08 4d 8b 28 49 8b 40 10 4d 85 ed 0f 84 76 01 00 00 48 85 c0 0f 84 6d 01 00 00 49 63 44 24 20 49 8b 3c 24 48 8d 4a 01 <49> 8b 5c 05 00 4c 89 e8 65 48 0f c7 0f 0f 94 c0 84 c0 74 b6 49 
Jun 16 10:58:09 server1 kernel: [79178.951372] RIP  [<ffffffff811d7278>] kmem_cache_alloc+0x78/0x210
Jun 16 10:58:09 server1 kernel: [79178.951374]  RSP <ffff8803fedc33f8>
Jun 16 10:58:09 server1 kernel: [79178.951378] ---[ end trace 3c48fcfab6da8ea9 ]---
```

```
Jun 16 10:57:51 server1 kernel: [79161.324285] general protection fault: 0000 [#10] SMP 
Jun 16 10:57:51 server1 kernel: [79161.324299] Modules linked in: ib_iser rdma_cm iw_cm ib_cm ib_sa ib_mad ib_core ib_addr iscsi_tcp libiscsi_tcp libiscsi scsi_transport_iscsi snd_hda_codec_hdmi crc32_pclmul aesni_intel aes_x86_64 lrw gf128mul glue_helper ablk_helper ghash_clmulni_intel cryptd i915 eeepc_wmi asus_wmi sparse_keymap nfsd snd_hda_codec_realtek auth_rpcgss kvm snd_hda_codec_generic nfs_acl mxm_wmi nfs snd_hda_intel coretemp drm_kms_helper crct10dif_pclmul snd_hda_controller drm intel_powerclamp lockd intel_rapl snd_hda_codec iosf_mbi dm_multipath snd_hwdep x86_pkg_temp_thermal scsi_dh grace snd_pcm snd_timer serio_raw lpc_ich mei_me snd shpchp soundcore i2c_algo_bit mei sunrpc 8250_fintek wmi tpm_infineon lp mac_hid video acpi_pad fscache parport zfs(POE) zunicode(POE) zcommon(POE) znvpair(POE) spl(OE) zavl(POE) psmouse ahci r8169 libahci mii dm_mirror dm_region_hash dm_log
Jun 16 10:57:51 server1 kernel: [79161.324302] CPU: 1 PID: 1717 Comm: nfsd Tainted: P      D    OE  3.19.0-031900-generic #201504091832
Jun 16 10:57:51 server1 kernel: [79161.324302] Hardware name: ASUS All Series/Z97-K, BIOS 2604 05/20/2015
Jun 16 10:57:51 server1 kernel: [79161.324303] task: ffff8800d3839d70 ti: ffff8803feee4000 task.ti: ffff8803feee4000
Jun 16 10:57:51 server1 kernel: [79161.324305] RIP: 0010:[<ffffffff811d7278>]  [<ffffffff811d7278>] kmem_cache_alloc+0x78/0x210
Jun 16 10:57:51 server1 kernel: [79161.324306] RSP: 0018:ffff8803feee7558  EFLAGS: 00010286
Jun 16 10:57:51 server1 kernel: [79161.324307] RAX: 0000000000000000 RBX: 0000000000004210 RCX: 00000000060ba345
Jun 16 10:57:51 server1 kernel: [79161.324307] RDX: 00000000060ba344 RSI: 0000000000004210 RDI: 000000000001dbd0
Jun 16 10:57:51 server1 kernel: [79161.324307] RBP: ffff8803feee75a8 R08: ffff88041fa9dbd0 R09: 0000000000020000
Jun 16 10:57:51 server1 kernel: [79161.324308] R10: ffff880036831868 R11: 0000000000000000 R12: ffff8800d357a700
Jun 16 10:57:51 server1 kernel: [79161.324308] R13: ffef88029dc16040 R14: ffff8800d357a700 R15: ffffffffc01bbd59
Jun 16 10:57:51 server1 kernel: [79161.324309] FS:  0000000000000000(0000) GS:ffff88041fa80000(0000) knlGS:0000000000000000
Jun 16 10:57:51 server1 kernel: [79161.324309] CS:  0010 DS: 0000 ES: 0000 CR0: 0000000080050033
Jun 16 10:57:51 server1 kernel: [79161.324309] CR2: 00007fdb0fa8c000 CR3: 0000000001c15000 CR4: 00000000001407e0
Jun 16 10:57:51 server1 kernel: [79161.324310] Stack:
Jun 16 10:57:51 server1 kernel: [79161.324311]  ffff8803feee7578 ffff88041fa13e40 ffff88041fa13eb8 00004210feee75d0
Jun 16 10:57:51 server1 kernel: [79161.324311]  ffffffff8101358e 0000000000000000 0000000000000000 0000000000000000
Jun 16 10:57:51 server1 kernel: [79161.324312]  ffff8800d357a700 ffff8800d3839d70 ffff8803feee7668 ffffffffc01bbd59
Jun 16 10:57:51 server1 kernel: [79161.324312] Call Trace:
Jun 16 10:57:51 server1 kernel: [79161.324315]  [<ffffffff8101358e>] ? __switch_to+0xbe/0x5b0
Jun 16 10:57:51 server1 kernel: [79161.324320]  [<ffffffffc01bbd59>] spl_kmem_cache_alloc+0xb9/0x7c0 [spl]
Jun 16 10:57:51 server1 kernel: [79161.324321]  [<ffffffff811d739d>] ? kmem_cache_alloc+0x19d/0x210
Jun 16 10:57:51 server1 kernel: [79161.324342]  [<ffffffffc03182dd>] zio_create+0x3d/0x500 [zfs]
Jun 16 10:57:51 server1 kernel: [79161.324360]  [<ffffffffc0318e9c>] zio_read+0xbc/0xe0 [zfs]
Jun 16 10:57:51 server1 kernel: [79161.324368]  [<ffffffffc026ae20>] ? arc_adapt_thread+0x210/0x210 [zfs]
Jun 16 10:57:51 server1 kernel: [79161.324375]  [<ffffffffc026ae20>] ? arc_adapt_thread+0x210/0x210 [zfs]
Jun 16 10:57:51 server1 kernel: [79161.324382]  [<ffffffffc026d83b>] arc_read+0x5ab/0xa60 [zfs]
Jun 16 10:57:51 server1 kernel: [79161.324391]  [<ffffffffc0275308>] dbuf_read+0x2a8/0x950 [zfs]
Jun 16 10:57:51 server1 kernel: [79161.324401]  [<ffffffffc027da93>] dmu_buf_hold_array_by_dnode+0x123/0x4d0 [zfs]
Jun 16 10:57:51 server1 kernel: [79161.324411]  [<ffffffffc027df1d>] dmu_buf_hold_array+0x5d/0x80 [zfs]
Jun 16 10:57:51 server1 kernel: [79161.324421]  [<ffffffffc027f5c1>] dmu_read_uio+0x41/0xe0 [zfs]
Jun 16 10:57:51 server1 kernel: [79161.324438]  [<ffffffffc030339c>] zfs_read+0x14c/0x440 [zfs]
Jun 16 10:57:51 server1 kernel: [79161.324456]  [<ffffffffc031f87a>] zpl_aio_read+0xda/0x130 [zfs]
Jun 16 10:57:51 server1 kernel: [79161.324458]  [<ffffffff811f4158>] do_sync_readv_writev+0x48/0x80
Jun 16 10:57:51 server1 kernel: [79161.324459]  [<ffffffff811f58d2>] do_readv_writev+0x1e2/0x2e0
Jun 16 10:57:51 server1 kernel: [79161.324461]  [<ffffffff813548e8>] ? ima_file_check+0x18/0x20
Jun 16 10:57:51 server1 kernel: [79161.324465]  [<ffffffffc085e1b5>] ? nfsd_open+0x145/0x1e0 [nfsd]
Jun 16 10:57:51 server1 kernel: [79161.324466]  [<ffffffff811f5a48>] vfs_readv+0x38/0x40
Jun 16 10:57:51 server1 kernel: [79161.324468]  [<ffffffffc085e333>] nfsd_readv+0x53/0x80 [nfsd]
Jun 16 10:57:51 server1 kernel: [79161.324471]  [<ffffffffc085e68f>] nfsd_read+0x6f/0xc0 [nfsd]
Jun 16 10:57:51 server1 kernel: [79161.324473]  [<ffffffff81099204>] ? groups_free+0x54/0x60
Jun 16 10:57:51 server1 kernel: [79161.324476]  [<ffffffffc0867780>] nfsd3_proc_read+0xc0/0x160 [nfsd]
Jun 16 10:57:51 server1 kernel: [79161.324479]  [<ffffffffc0858e55>] nfsd_dispatch+0xe5/0x230 [nfsd]
Jun 16 10:57:51 server1 kernel: [79161.324487]  [<ffffffffc054d764>] svc_process_common+0x324/0x680 [sunrpc]
Jun 16 10:57:51 server1 kernel: [79161.324494]  [<ffffffffc054de23>] svc_process+0x113/0x1d0 [sunrpc]
Jun 16 10:57:51 server1 kernel: [79161.324496]  [<ffffffffc08589c7>] nfsd+0x117/0x190 [nfsd]
Jun 16 10:57:51 server1 kernel: [79161.324499]  [<ffffffffc08588b0>] ? nfsd_destroy+0x80/0x80 [nfsd]
Jun 16 10:57:51 server1 kernel: [79161.324500]  [<ffffffff81095d59>] kthread+0xc9/0xe0
Jun 16 10:57:51 server1 kernel: [79161.324501]  [<ffffffff81095c90>] ? flush_kthread_worker+0x90/0x90
Jun 16 10:57:51 server1 kernel: [79161.324504]  [<ffffffff817d2cbc>] ret_from_fork+0x7c/0xb0
Jun 16 10:57:51 server1 kernel: [79161.324505]  [<ffffffff81095c90>] ? flush_kthread_worker+0x90/0x90
Jun 16 10:57:51 server1 kernel: [79161.324513] Code: 7e 49 8b 50 08 4d 8b 28 49 8b 40 10 4d 85 ed 0f 84 76 01 00 00 48 85 c0 0f 84 6d 01 00 00 49 63 44 24 20 49 8b 3c 24 48 8d 4a 01 <49> 8b 5c 05 00 4c 89 e8 65 48 0f c7 0f 0f 94 c0 84 c0 74 b6 49 
Jun 16 10:57:51 server1 kernel: [79161.324514] RIP  [<ffffffff811d7278>] kmem_cache_alloc+0x78/0x210
Jun 16 10:57:51 server1 kernel: [79161.324514]  RSP <ffff8803feee7558>
Jun 16 10:57:51 server1 kernel: [79161.324515] ---[ end trace 3c48fcfab6da8ea8 ]---
```

```
Jun 16 10:57:51 server1 kernel: [79161.318125] general protection fault: 0000 [#9] SMP 
Jun 16 10:57:51 server1 kernel: [79161.318144] Modules linked in: ib_iser rdma_cm iw_cm ib_cm ib_sa ib_mad ib_core ib_addr iscsi_tcp libiscsi_tcp libiscsi scsi_transport_iscsi snd_hda_codec_hdmi crc32_pclmul aesni_intel aes_x86_64 lrw gf128mul glue_helper ablk_helper ghash_clmulni_intel cryptd i915 eeepc_wmi asus_wmi sparse_keymap nfsd snd_hda_codec_realtek auth_rpcgss kvm snd_hda_codec_generic nfs_acl mxm_wmi nfs snd_hda_intel coretemp drm_kms_helper crct10dif_pclmul snd_hda_controller drm intel_powerclamp lockd intel_rapl snd_hda_codec iosf_mbi dm_multipath snd_hwdep x86_pkg_temp_thermal scsi_dh grace snd_pcm snd_timer serio_raw lpc_ich mei_me snd shpchp soundcore i2c_algo_bit mei sunrpc 8250_fintek wmi tpm_infineon lp mac_hid video acpi_pad fscache parport zfs(POE) zunicode(POE) zcommon(POE) znvpair(POE) spl(OE) zavl(POE) psmouse ahci r8169 libahci mii dm_mirror dm_region_hash dm_log
Jun 16 10:57:51 server1 kernel: [79161.318147] CPU: 1 PID: 1723 Comm: nfsd Tainted: P      D    OE  3.19.0-031900-generic #201504091832
Jun 16 10:57:51 server1 kernel: [79161.318148] Hardware name: ASUS All Series/Z97-K, BIOS 2604 05/20/2015
Jun 16 10:57:51 server1 kernel: [79161.318149] task: ffff8804004309d0 ti: ffff8803fe4f0000 task.ti: ffff8803fe4f0000
Jun 16 10:57:51 server1 kernel: [79161.318154] RIP: 0010:[<ffffffff811d7278>]  [<ffffffff811d7278>] kmem_cache_alloc+0x78/0x210
Jun 16 10:57:51 server1 kernel: [79161.318154] RSP: 0018:ffff8803fe4f36e8  EFLAGS: 00010286
Jun 16 10:57:51 server1 kernel: [79161.318155] RAX: 0000000000000000 RBX: 0000000000004210 RCX: 00000000060ba345
Jun 16 10:57:51 server1 kernel: [79161.318155] RDX: 00000000060ba344 RSI: 0000000000004210 RDI: 000000000001dbd0
Jun 16 10:57:51 server1 kernel: [79161.318155] RBP: ffff8803fe4f3738 R08: ffff88041fa9dbd0 R09: 0000000000000000
Jun 16 10:57:51 server1 kernel: [79161.318156] R10: ffff880036831868 R11: ffff880403bb4500 R12: ffff8800d357a700
Jun 16 10:57:51 server1 kernel: [79161.318156] R13: ffef88029dc16040 R14: ffff8800d357a700 R15: ffffffffc01bbd59
Jun 16 10:57:51 server1 kernel: [79161.318157] FS:  0000000000000000(0000) GS:ffff88041fa80000(0000) knlGS:0000000000000000
Jun 16 10:57:51 server1 kernel: [79161.318157] CS:  0010 DS: 0000 ES: 0000 CR0: 0000000080050033
Jun 16 10:57:51 server1 kernel: [79161.318158] CR2: 00007fdb0fa8c000 CR3: 0000000001c15000 CR4: 00000000001407e0
Jun 16 10:57:51 server1 kernel: [79161.318158] Stack:
Jun 16 10:57:51 server1 kernel: [79161.318159]  0000000000000000 ffff8800366769a0 ffff8803fe4f37a8 00004210c0273633
Jun 16 10:57:51 server1 kernel: [79161.318160]  0000000000000202 0000000000000000 0000000000000000 0000000000000000
Jun 16 10:57:51 server1 kernel: [79161.318161]  ffff8800d357a700 ffff8804004309d0 ffff8803fe4f37f8 ffffffffc01bbd59
Jun 16 10:57:51 server1 kernel: [79161.318161] Call Trace:
Jun 16 10:57:51 server1 kernel: [79161.318168]  [<ffffffffc01bbd59>] spl_kmem_cache_alloc+0xb9/0x7c0 [spl]
Jun 16 10:57:51 server1 kernel: [79161.318171]  [<ffffffff817d0be6>] ? mutex_lock+0x16/0x37
Jun 16 10:57:51 server1 kernel: [79161.318185]  [<ffffffffc0272671>] ? dbuf_find+0x1d1/0x1e0 [zfs]
Jun 16 10:57:51 server1 kernel: [79161.318190]  [<ffffffffc01bac40>] ? spl_kmem_zalloc+0xf0/0x1b0 [spl]
Jun 16 10:57:51 server1 kernel: [79161.318213]  [<ffffffffc03182dd>] zio_create+0x3d/0x500 [zfs]
Jun 16 10:57:51 server1 kernel: [79161.318233]  [<ffffffffc03189d1>] zio_null+0x61/0x70 [zfs]
Jun 16 10:57:51 server1 kernel: [79161.318252]  [<ffffffffc03189fe>] zio_root+0x1e/0x20 [zfs]
Jun 16 10:57:51 server1 kernel: [79161.318263]  [<ffffffffc027da25>] dmu_buf_hold_array_by_dnode+0xb5/0x4d0 [zfs]
Jun 16 10:57:51 server1 kernel: [79161.318274]  [<ffffffffc027df1d>] dmu_buf_hold_array+0x5d/0x80 [zfs]
Jun 16 10:57:51 server1 kernel: [79161.318284]  [<ffffffffc027f5c1>] dmu_read_uio+0x41/0xe0 [zfs]
Jun 16 10:57:51 server1 kernel: [79161.318303]  [<ffffffffc030339c>] zfs_read+0x14c/0x440 [zfs]
Jun 16 10:57:51 server1 kernel: [79161.318321]  [<ffffffffc031f87a>] zpl_aio_read+0xda/0x130 [zfs]
Jun 16 10:57:51 server1 kernel: [79161.318324]  [<ffffffff811f4158>] do_sync_readv_writev+0x48/0x80
Jun 16 10:57:51 server1 kernel: [79161.318325]  [<ffffffff811f58d2>] do_readv_writev+0x1e2/0x2e0
Jun 16 10:57:51 server1 kernel: [79161.318327]  [<ffffffff813548e8>] ? ima_file_check+0x18/0x20
Jun 16 10:57:51 server1 kernel: [79161.318331]  [<ffffffffc085e1b5>] ? nfsd_open+0x145/0x1e0 [nfsd]
Jun 16 10:57:51 server1 kernel: [79161.318332]  [<ffffffff811f5a48>] vfs_readv+0x38/0x40
Jun 16 10:57:51 server1 kernel: [79161.318335]  [<ffffffffc085e333>] nfsd_readv+0x53/0x80 [nfsd]
Jun 16 10:57:51 server1 kernel: [79161.318338]  [<ffffffffc085e68f>] nfsd_read+0x6f/0xc0 [nfsd]
Jun 16 10:57:51 server1 kernel: [79161.318341]  [<ffffffff81099204>] ? groups_free+0x54/0x60
Jun 16 10:57:51 server1 kernel: [79161.318344]  [<ffffffffc0867780>] nfsd3_proc_read+0xc0/0x160 [nfsd]
Jun 16 10:57:51 server1 kernel: [79161.318347]  [<ffffffffc0858e55>] nfsd_dispatch+0xe5/0x230 [nfsd]
Jun 16 10:57:51 server1 kernel: [79161.318356]  [<ffffffffc054d764>] svc_process_common+0x324/0x680 [sunrpc]
Jun 16 10:57:51 server1 kernel: [79161.318362]  [<ffffffffc054de23>] svc_process+0x113/0x1d0 [sunrpc]
Jun 16 10:57:51 server1 kernel: [79161.318365]  [<ffffffffc08589c7>] nfsd+0x117/0x190 [nfsd]
Jun 16 10:57:51 server1 kernel: [79161.318367]  [<ffffffffc08588b0>] ? nfsd_destroy+0x80/0x80 [nfsd]
Jun 16 10:57:51 server1 kernel: [79161.318368]  [<ffffffff81095d59>] kthread+0xc9/0xe0
Jun 16 10:57:51 server1 kernel: [79161.318370]  [<ffffffff81095c90>] ? flush_kthread_worker+0x90/0x90
Jun 16 10:57:51 server1 kernel: [79161.318372]  [<ffffffff817d2cbc>] ret_from_fork+0x7c/0xb0
Jun 16 10:57:51 server1 kernel: [79161.318373]  [<ffffffff81095c90>] ? flush_kthread_worker+0x90/0x90
Jun 16 10:57:51 server1 kernel: [79161.318381] Code: 7e 49 8b 50 08 4d 8b 28 49 8b 40 10 4d 85 ed 0f 84 76 01 00 00 48 85 c0 0f 84 6d 01 00 00 49 63 44 24 20 49 8b 3c 24 48 8d 4a 01 <49> 8b 5c 05 00 4c 89 e8 65 48 0f c7 0f 0f 94 c0 84 c0 74 b6 49 
Jun 16 10:57:51 server1 kernel: [79161.318383] RIP  [<ffffffff811d7278>] kmem_cache_alloc+0x78/0x210
Jun 16 10:57:51 server1 kernel: [79161.318383]  RSP <ffff8803fe4f36e8>
Jun 16 10:57:51 server1 kernel: [79161.318416] ---[ end trace 3c48fcfab6da8ea7 ]---
```

**Comment by behlendorf** [comment]
Closing as stale.  If you're still able to reproduce this issue with 0.6.5.8 or newer open a new issue.
